### PR TITLE
GEODE-2428: Adding support of Serialization/deserialization of LinkedHashMap

### PR DESCRIPTION
**Pull request by davinash**
Added support for writeLinkedHashMap and readLinkedHashMap and the test for the same.

### CODE REVIEW

File: geode-core/src/main/java/org/apache/geode/DataSerializer.java [hunk 1]
```diff
@@ -24,23 +24,7 @@
...
-import java.util.LinkedHashSet;
-import java.util.LinkedList;
-import java.util.Map;
-import java.util.Properties;
-import java.util.Set;
-import java.util.Stack;
-import java.util.TreeMap;
-import java.util.TreeSet;
-import java.util.Vector;
+import java.util.*;
```
Comment (metatype) [inline geode-core/src/main/java/org/apache/geode/DataSerializer.java]: I'd prefer to leave this as explicit import statements instead of wildcard import.

**Comment by metatype** [comment]
This looks good to me, @bschuchardt what do you think?

File: geode-core/src/main/java/org/apache/geode/DataSerializer.java (hunk 1 above)
Comment (bschuchardt) [inline geode-core/src/main/java/org/apache/geode/DataSerializer.java]: The intellij formatter does this to you unless you modify it

**Comment by davinash** [comment]
Thanks @metatype and @bschuchardt 
I will fix import issue and check-in the changes to develop, once all the checks have passed.

**Comment by davinash** [comment]
Closing, merged to develop

**Pull request closed without merging**